frontend/src/pages/Inbox/components/EmailListItem.js

```diff
@@ -70,3 +68,8 @@ export default function EmailListItem({
     </Route>
   );
 }
+
+export default connect(
+  null,
+  { markThreadAsActive }
```

Comment:
Why not use redux hook here?

Reply:
I think I already have a task to refactor this file. That's why I need that all PRs be approved to see if we miss anything. Because is start become confusing.